### ISSUE TITLE
[PS1]How to use the custome BIOS？

### ISSUE THREAD

**Issue by Legend-Huang** [libretro/RetroArch#11358]
There is some edited games must run with special BIOS，not official BIOS.How to custome it?

**Comment by i30817** [comment]
Compile beetle yourself and pray it works. Most emulator authors are not going to support homebrew bioses.

**Comment by hizzlekizzle** [comment]
I think if you just rename it to one of the ones it's looking for, it should try to load it even if the checksum doesn't match, right? We can't guarantee that it'll work, but I think the core should at least attempt it.

**Comment by i30817** [comment]
Eh... makes sense. But it's not ergonomic when the person wants to play another game.

**Comment by hizzlekizzle** [comment]
Yeah, it's definitely cumbersome as it is, and I wouldn't want to support swapping them out or maintaining per-game BIOS, etc. to improve it.

**Comment by CaiMiao** [comment]
I think warn users with clear declaration of no technical support, when they attempt to use a mismatching BIOS, should be fine?
It is sad that some platform (WiiU or 3DS) only receive libretro port of emulators, and some fan-translation (hacks) require custom BIOS for proper glyph render (Chinese font is huge) are unplayable on those platform.

**Comment by sonninnos** [comment]
In any case this is a core issue and not RA.
